.vscode/extensions.json

```diff
@@ -0,0 +1,3 @@
+{
+    "recommendations": ["swiftlang.swift-vscode", "Vapor.vapor-vscode"]
```

Comment:
`"usernamehw.errorlens"` is also nice if we want to get a bit opinionated cc @gwynne @0xTim @ptoffy

Reply:
I think lets stick to ones we have (at least indirect) control over

Reply:
I concur.